packages/manager/modules/sms/src/sms/batches/history/delete/delete.controller.js

```diff
@@ -0,0 +1,35 @@
+export default class TelecomSmsBatchesHistoryDeleteCtrl {
+  /* @ngInject */
+  constructor($http, Alerter, $translate, SmsService) {
+    this.isLoading = false;
+    this.$http = $http;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+    this.SmsService = SmsService;
+  }
+
+  delete() {
+    this.isLoading = true;
+
+    return this.SmsService.deleteBatchHistory(this.serviceName, this.batchId)
+      .then(() => {
+        this.Alerter.success(
+          this.$translate.instant(
+            'sms_batches_history_delete_confirmation_message',
+          ),
+          'telecom.main.alerter',
+        );
+        this.goBack(true);
+      })
+      .catch(() => {
+        this.Alerter.error(
+          this.$translate.instant('sms_batches_history_delete_error_message'),
+          'telecom.main.alerter',
+        );
```

Comment:
**nitpick:** No need to inject the `ng-ovh-utils`. In Telecom universe, you can use `TucToast` to display info/error instead of Alerter.

Reply:
Done